arviz/plots/densityplot.py

```diff
@@ -37,42 +37,40 @@ def plot_density(
 
     Parameters
     ----------
-    data : object or iterable of object
+    data : InferenceData or iterable of object
         Any object that can be converted to an :class:`arviz.InferenceData` object, or an Iterator
         returning a sequence of such objects.
-        Refer to documentation of :func:`arviz.convert_to_dataset` for details about such objects.
-    group : str, optional
-        Specifies which :class:`arviz.InferenceData` group should be plotted.
-        Defaults to 'posterior'.
-        Alternative values include 'prior' and any other strings used as dataset keys in the
-        :class:`arviz.InferenceData`.
-    data_labels : list of str, optional
+        Refer to documentation of :func:`arviz.convert_to_dataset` for details.
+    group : {"posterior", "prior"}, default "posterior"
+        Specifies which InferenceData group should be plotted. If "posterior", then the values
+        in `posterior_predictive` group are compared to the ones in `observed_data`, if "prior" then
+        the same comparison happens, but with the values in `prior_predictive` group.
+    data_labels : list of str, default None
         List with names for the datasets passed as "data." Useful when plotting more than one
-        dataset.  Must be the same shape as the data parameter.  Defaults to None.
+        dataset.  Must be the same shape as the data parameter.
     var_names : list of str, optional
-        List of variables to plot.  If multiple datasets are supplied and var_names is not None,
+        List of variables to plot.  If multiple datasets are supplied and `var_names` is not None,
         will print the same set of variables for each dataset.  Defaults to None, which results in
         all the variables being plotted.
-    filter_vars : {None, "like", "regex"}, default=None
-        If `None` (default), interpret var_names as the real variables names. If "like",
-        interpret var_names as substrings of the real variables names. If "regex",
-        interpret var_names as regular expressions on the real variables names. A la
-        ``pandas.filter``.
+    filter_vars : {None, "like", "regex"}, default None
+        If `None` (default), interpret `var_names` as the real variables names. If "like",
+        interpret `var_names` as substrings of the real variables names. If "regex",
+        interpret `var_names` as regular expressions on the real variables names. See
+        the :ref:`this section <common_filter_vars>` for usage examples.
     combine_dims : set_like of str, optional
         List of dimensions to reduce. Defaults to reducing only the "chain" and "draw" dimensions.
         See the :ref:`this section <common_combine_dims>` for usage examples.
     transform : callable
-        Function to transform data (defaults to None i.e. the identity function)
-    hdi_prob : float
+        Function to transform data (defaults to `None` i.e. the identity function).
+    hdi_prob : float, default 0.94
```

Comment:
haven't checked the code, but the default should be the `stats.hdi_prob` rcParam like in other hdi_prob arguments